dockers/docker-snmp/snmpd.conf.j2

```diff
@@ -34,16 +34,56 @@ view   systemonly  included   .1.3.6.1.2.1.1
 view   systemonly  included   .1.3.6.1.2.1.25.1
 
                                                  #  Default access to basic system info
-{% if snmp_rocommunities %}
+{% if SNMP_COMMUNITY is defined %}
+
+{% for community in SNMP_COMMUNITY %}
+{% if SNMP_COMMUNITY[community]['TYPE'] == 'RO' %}
+rocommunity {{ community }}
+rocommunity6 {{ community }}
+{% elif SNMP_COMMUNITY[community]['TYPE'] == 'RW' %}
+rwcommunity {{ community }}
+rwcommunity6 {{ community }}
+{% endif %}
+{% endfor %}
+{% endif %}
+
+{% if SNMP_COMMUNITY is defined and snmp_rocommunities is defined %}
```

Comment:
I understand you want to mix community strings from both source. Another strategy is just simply ignore snmp_rocommunities in yml file if SNMP_COMMUNITY is available in Redis.
We don't have strong use case to mix them, if user want to use Redis, user will add all the community strings (even old ones in yml file) to Redis.
So the preference is just a protection for corner cases.

Reply:
updated to only use configdb.  That should address this.